netmaster/master/endpointGroup.go

```diff
@@ -80,13 +80,15 @@ func CreateEndpointGroup(tenantName, networkName, groupName string, epgID int) e
 	}
 
 	// params for docker network
-	subnetCIDR := fmt.Sprintf("%s/%d", nwCfg.SubnetIP, nwCfg.SubnetLen)
+	if GetClusterMode() == "docker" {
```

Comment:
Better way to do this might be to create mgmtfn abstraction here. so that we can move docker/swarm specific things into docker mgmtfn.
We can do this as a subsequent PR too.